FluidNC/src/Motors/TMC5160Driver.cpp

```diff
@@ -29,7 +29,17 @@ namespace MotorDrivers {
         TrinamicBase::config_motor();
     }
 
-    bool TMC5160Driver::test() { return TrinamicBase::reportTest(tmc5160->test_connection()); }
+    bool TMC5160Driver::test() {
```

Comment:
From https://www.trinamic.com/fileadmin/assets/Products/ICs_Documents/TMC5160A_Datasheet_Rev1.14.pdf  page 33:
![image](https://user-images.githubusercontent.com/4861133/188546680-d571ed07-de4b-447b-8dab-c3c884c96a29.png)

We only support SPI mode for TMC5160s so I don't think this particular instance of the change will work.